1_fetch/src/write_data.R

```diff
@@ -0,0 +1,7 @@
+library(readr)
```

Comment:
I think the code should run without the explicit call to `readr` here since the tidyverse collection of packages is included in `tar_option_set(...)` within the `_targets.R` file.

Reply:
Oh. Cool! I did not realize that.